shared/components/pages/take_action/action/action.test.js

```diff
@@ -1,14 +1,14 @@
 /* global describe it expect*/
 
 import TestUtils from 'react-addons-test-utils';
-import React from 'react';
+import wrapWithContext from '../../../application/application.component.mock';
 
 import Action from './action.component';
 
 describe('Action component', () => {
   it('renders without problems', (done) => {
-    const action = TestUtils.renderIntoDocument(React.createElement(Action));
-    expect(action.state).toEqual({});
+    const action = TestUtils.renderIntoDocument(wrapWithContext(Action));
```

Comment:
Currently throws an error warning for not having specified a required prop. Easy fix with
```js
const action = TestUtils.renderIntoDocument(
      wrapWithContext(Action, {}, { is_assumption: false }));
```